### DISCUSSION TITLE
Measure voltage by one of the clamps?

### DISCUSSION THREAD

**Discussion by karpeychik** [emporia-vue-local/esphome#199] [Q&A]
I was doing some work in my panel and noticed that my septic control breaker has turned off. I have no idea when this happened and this is quite scary.

Emporia can measure voltage in main, can it do in in small clamps? I was hoping to set up monitoring to alarm myself that there is no voltage in the line.

**Comment by flaviut** [comment]
Clamps can't measure voltage. If you have an extra voltage wire in your harness, you can hook that up to the breaker if you'd like.

Instead of measuring voltage, I'd suggest setting up an automation based on current draw. If your equipment draws an amount of current that is out of line with typical current (too high, too low), then alert on that. This works with faults down the line as well, like a cut wire or a stalled pump, not just upstream faults.
